lib/immutable-struct.rb

```diff
@@ -6,7 +6,7 @@
 # will be evaluated as if it were inside a class definition, allowing you
 # to add methods, include or extend modules, or do whatever else you want.
 class ImmutableStruct
-  VERSION='2.1.2' #:nodoc:
+  VERSION='2.1.3' #:nodoc:
```

Comment:
Can you move this back?  We'll bump the version when we do the release (and I think this would be a minor since this is a new feature).